test/libsolidity/util/SoltestTypes.h

```diff
@@ -42,6 +42,7 @@ namespace solidity::frontend::test
 	T(Comma,  ",", 0)                  \
 	T(Period, ".", 0)                  \
 	T(Arrow, "->", 0)                  \
+	T(Effect, "- ", 0)                 \
```

Comment:
Does it make sense to include whitespace in a token? Will that even work?

Reply:
Yep, let's not use whitespace here.

Reply:
Also please call it `Minus or something.

Reply:
Continued in https://github.com/ethereum/solidity/pull/11048#discussion_r631242943.